<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>658</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>658</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>658</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PokeLids</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>10</y>
      <width>521</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPrefectures">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Prefecture</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="area1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>141</y>
      <width>136</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Hokkaido · Tohoku</string>
    </property>
   </widget>
   <widget class="QPushButton" name="area2">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>259</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Kanto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="area3">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>255</y>
      <width>63</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Chubu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="area4">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>281</y>
      <width>55</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Kinki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="area5">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>222</y>
      <width>136</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Chugoku · Shikoku</string>
    </property>
   </widget>
   <widget class="QPushButton" name="area6">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>287</y>
      <width>132</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Kyushu · Okinawa</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="obtained">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtained</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotomap">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View on Map!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>592</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarPrefecture">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>368</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelCity">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarCity">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>368</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelTotal">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarTotal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>598</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>670</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>600</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetIds">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefecture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetCities">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabelPrefecture_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokémon:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoprefecturemap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefecture Map</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>map</zorder>
   <zorder>area1</zorder>
   <zorder>comboBoxPrefectures</zorder>
   <zorder>area2</zorder>
   <zorder>area3</zorder>
   <zorder>area4</zorder>
   <zorder>area5</zorder>
   <zorder>area6</zorder>
   <zorder>imageLid</zorder>
   <zorder>obtained</zorder>
   <zorder>gotomap</zorder>
   <zorder>exit</zorder>
   <zorder>progressBarPrefecture</zorder>
   <zorder>progressLabelPrefecture</zorder>
   <zorder>progressLabelCity</zorder>
   <zorder>progressBarCity</zorder>
   <zorder>progressLabelTotal</zorder>
   <zorder>progressBarTotal</zorder>
   <zorder>listWidget</zorder>
   <zorder>progressLabelPrefecture_4</zorder>
   <zorder>listWidgetIds</zorder>
   <zorder>progressLabelPrefecture_2</zorder>
   <zorder>progressLabelPrefecture_3</zorder>
   <zorder>listWidgetCities</zorder>
   <zorder>progressLabelPrefecture_5</zorder>
   <zorder>progressLabelPrefecture_6</zorder>
   <zorder>progressLabelPrefecture_7</zorder>
   <zorder>gotoprefecturemap</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Reset_DB"/>
    <addaction name="action_Expert_Mode"/>
   </widget>
   <addaction name="menu_Settings"/>
  </widget>
  <action name="action_Reset_DB">
   <property name="text">
    <string>&amp;Reset DB</string>
   </property>
  </action>
  <action name="action_Expert_Mode">
   <property name="text">
    <string>&amp;Expert Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
